modules/core/src/test/java/org/apache/ignite/internal/processors/database/BPlusTreeSelfTest.java

```diff
@@ -208,10 +208,22 @@ protected ReuseList createReuseList(int cacheId, PageMemory pageMem, long rootId
 
                 try {
                     asyncRunFut.cancel();
-                    asyncRunFut.get(60000);
+                    asyncRunFut.get(TimeUnit.MINUTES.toMillis(1));
```

Comment:
Why do you need this get if you added new loop?

Reply:
Loop as an additional protection, in general, I think waiting on the future should be enough.